mediacapture-streams/enumerating-local-media-devices/api-present.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+<title>enumerateDevices: test that enumerateDevices is present</title>
+<link rel="author" title="Dr Alex Gouaillard" href="mailto:agouaillard@gmail.com"/>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#enumerating-devices">
+<meta name='assert' content='Check that the enumerateDevices() method is present.'/>
+<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
```

Comment:
I understand that we don't need (and thus shouldn't) link to the stylesheet, since it's added automatically by testharness.js.